agent/command/s3_put_test.go

```diff
@@ -501,7 +502,11 @@ func TestPreservePath(t *testing.T) {
 		"remote/myWebsite/assets/images/image1": false,
 		"remote/myWebsite/assets/images/image2": false,
 	}
+	grip.Infof("expected: %s", expected)
+	grip.Infof("received: ")
 	for it.Next(ctx) {
+		grip.Infof("\n: %s", it.Item().Name())
```

Comment:
Is this meant to be committed or just for temporary testing? We don't normally log in tests because you can typically add assert/require statements that will do the equivalent, without the logging noise.

I guess, what kind of information do we get out of the logs that we couldn't get from the assert/require statements?

Reply:
This is just for debugging. It's passing in patches but not the repotracker